### PR TITLE
CSV and README updates

### PR DESCRIPTION
**Pull request by swati-nair**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,142 @@
...
+## Supported platforms
 
-## Operating Systems
+Red Hat OpenShift Container Platform 4.2 or newer installed on one of the following platforms:
 
-- Linux amd64
-- RHEL amd64
-- RHEL ppc64le
-- RHEL s390x
+Linux x86_64
```
Comment (tthorpe2) [inline README.md:13]: Can you make this a bullet list?

File: README.md [hunk 2]
```diff
@@ -1,57 +1,142 @@
...
+Linux on Power (ppc64le)
+Linux on IBM Z and LinuxONE
 
 ## Operator versions
 
 - 3.5.0
+- 3.6.0
+  Features in 3.6.0 release: RHACM coexistence support, operator-to-operator upgrade, helm-to-operator upgrade, operand image reference using SHA digest, IBM certification.
```
Comment (tthorpe2) [inline README.md:21]: Let's drop this part -- 
"Features in 3.6.0 release: RHACM coexistence support, operator-to-operator upgrade, helm-to-operator upgrade, operand image reference using SHA digest, IBM certification."    
These are all plumbing items and not user features.

File: README.md [hunk 3]
```diff
@@ -1,57 +1,142 @@
...
 
-For installation and configuration, see the [IBM Cloud Platform Common Services documentation](http://ibm.biz/cpcsdocs).
+To install the operator with the IBM Common Services Operator follow the the installation and configuration instructions within the IBM Knowledge Center.
+
+- If you are using the operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak. For a list of IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+- If you are using the operator with an IBM Containerized Software, see the IBM Cloud Platform Common Services Knowledge Center [Installer documentation](http://ibm.biz/cpcs_opinstall).
+
+## SecurityContextConstraints Requirements
+
+The Common UI service supports running with the OpenShift Container Platform 4.3 default restricted Security Context Constraints (SCCs).
```
Comment (tthorpe2) [inline README.md:40]: Missed one change
replace    ---   The Common UI service supports
with         ---    The IBM certificate manager service

File: README.md (hunk 1 above)
Comment (swati-nair) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (swati-nair) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (swati-nair) [reply README.md]: Updated

**Pull request merged**